### ISSUE TITLE
Accessible User Profile

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#1551]
The User Profile dropdown in msVersion2 does not currently have alt text on the button.

This should be updated to contain the alt text "User Profile".

**Comment by abaranowska1** [comment]
@alexrcabral @rupeshparab 
I used Screen Reader Extension for Google Chrome and I see that after click avatar I cannot hear " User Profile". The screen reader read me only "name", "mail" from the dropdown and the text "Sign Out". 
![Selection_535](https://user-images.githubusercontent.com/61964176/101357953-78ef4d80-389a-11eb-8c0c-9db158fb31fa.png)

**Comment by rupeshparab** [comment]
On trying this on Voicover I can see that User Profile stays constant on opening the dropdown, @alexrcabral Do you think we should add a suffix of the dropdown status?

This the text in voiceover for each step:

Dropdown Closed:
![image](https://user-images.githubusercontent.com/6897505/101392993-6e5da400-38ec-11eb-8a9b-56930000d7b5.png)

Dropdown Opened:
![image](https://user-images.githubusercontent.com/6897505/101393027-774e7580-38ec-11eb-924b-13b60c7d680d.png)

Tab to inner item in list
![image](https://user-images.githubusercontent.com/6897505/101392936-5a19a700-38ec-11eb-92cc-c932e659a75d.png)

Tab to signout
![image](https://user-images.githubusercontent.com/6897505/101393042-7c132980-38ec-11eb-8e96-876bb26e72c4.png)

**Comment by alexrcabral** [comment]
@rupeshparab Thanks for the update. Yes I think adding a suffix to the user profile would be helpful. Ideally it should read similarly to the v1 profile button: "User profile, expanded, button" and "User profile, collapsed, button" to give clearer button feedback.

It would also help screen reader users to have it laid out similarly to v1 in differentiating between text items (inactionable) in the list and button items (actionable) in the list. I've broken out a separate issue (#1564)  for this.
